### PR TITLE
Add helpers to create metric/span monitoring using OpenCensus

### PR DESCRIPTION
**Pull request by AdamMagaluk**

### CODE REVIEW

File: perf/metrics.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import (
+	"fmt"
+	"log"
+
+	"go.opencensus.io/stats/view"
+	"go.uber.org/multierr"
+)
+
+// RegisterApplicationViews registers all the default views we may need for the application. gRPC, MongoDb, http, etc...
```
Comment (edaniels) [inline perf/metrics.go:11]: [supernit] MongoDb -> MongoDB; http -> HTTP

File: perf/metrics.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+// the exported data to the console.
+//
+// The intent is help new users familiarize themselves with the
+// capabilities of opencensus.
+//
+// This should NOT be used for production workloads.
+type PrintExporter struct{}
+
+// ExportView logs the view data.
+func (e *PrintExporter) ExportView(vd *view.Data) {
```
Comment (edaniels) [inline perf/metrics.go:38]: can you add somewhere in a readme or doc.go how to use all this stuff? either via GCP / a local server/viewer/printer?

File: perf/stackdriver.go [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+// NewCloudExporter creates a new Stackdriver (Cloud Monitoring) OpenCensus exporter with all options setup.
+func NewCloudExporter(ctx context.Context, logger golog.Logger) (Exporter, error) {
+	sdOpts := stackdriver.Options{
+		Context: ctx,
+		OnError: func(err error) {
+			logger.Errorw("opencensus stackdriver error", "error", err)
+		},
+		// ReportingInterval sets the frequency of reporting metrics to stackdriver backend.
+		ReportingInterval: 60 * time.Second,
+		MetricPrefix:      path.Join("app.viam.com", "opencensus"),
```
Comment (edaniels) [inline perf/stackdriver.go:39]: this is a public library so we can't use this value here. this should be usable by anyone with minimal work as a utility library on top of rpc.

**Review by edaniels — CHANGES_REQUESTED** [review]

File: perf/metrics.go (hunk 2 above)
Comment (AdamMagaluk) [reply perf/metrics.go]: I added a few examples to exporters_test.go

File: perf/stackdriver.go (hunk 3 above)
Comment (AdamMagaluk) [reply perf/stackdriver.go]: Ah good point. There's now a options struct passed in with it and the logger/context.

**Review by edaniels — APPROVED** [review]

**Pull request merged**